demo/Titanic/README.md

```diff
@@ -0,0 +1,61 @@
+# robotMBT Titanic demo
+
+## What is it?
+
+The purpose of this demo is to showcase the Model-Based Testing concepts available from the [robotMBT](https://github.com/JFoederer/robotframeworkMBT) library using a [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) style project. It is based on the principle of [specification by example](https://en.wikipedia.org/wiki/Specification_by_example), using _given-when-then_ style scenarios.
+
+Given-steps typically describe preconditions, i.e. state, but classically given-steps are implemented as actions to get to that desired precondition. Now, consider using [specification by example](https://en.wikipedia.org/wiki/Specification_by_example). If your specification is complete, and your examples are consistent, then any given-state must be reachable by operating the system within specification, following the examples. In this demo we use [robotMBT](https://github.com/JFoederer/robotframeworkMBT) to specify a complete story, on varying levels of detail, using small consise scenarios. Then we let [robotMBT](https://github.com/JFoederer/robotframeworkMBT) construct a complete storyline, so we don't have to worry about how to reach all the correct preconditions.
+
+Please keep in mind that the library and this demo are still in the early development phases and offered functionality is still limited. However, in good agile spirit, we still wanted to publish the results.
+
+## Contents of the demo - hit or miss?
+
+Two story lines can be played out in this demo, both composed from small specification scenarios. The `miss` scenario reveils the story as it was expected to happen. Titanic is not fazed by any iceberg and safely travels to New York. The `hit` scenario resembles the plot that made the history books.
+
+There are a total of 7 seven scenarios in this demo. Some are very high level, stating functional or performance requirements on Titanic's voyage as a whole. Others are zoomed in to more detail, defining what the expectations are for the vessel when encountering icebergs along the way. Two scenarios are mutually exclusive, but with either one a complete voyage can be specified when combining it with the other available scenarios.
+
+## Why the story of Titanic?
+
+It might seem odd at first, seeing the [story of Titanic](https://en.wikipedia.org/wiki/Sinking_of_the_Titanic) in a context that is typically used in more technical environments. However, the [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) process is mostly non-technical and using a topic like Titanic helps to prevent technical bias. Another important point is that we want to stick to writing and maintaining short, to-the-point scenarios. From these we want to compose larger scenarios, describing behaviour of complex systems, start to end, like telling a story. What better use for that than a well known story?
```

Comment:
Is the story typically used in more technical environments?

Comma after 'From these'

Reply:
The more technical context referred to is Behaviour-Driven Development or software engineering in general. Any suggestions on how to clarify this?